ProjFS.Linux/Scripts/Build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+CONFIGURATION=$1
+if [ -z $CONFIGURATION ]; then
+  CONFIGURATION=Debug
+fi
+
+SCRIPTDIR=$(dirname ${BASH_SOURCE[0]})
+SRCDIR=$SCRIPTDIR/../..
+ROOTDIR=$SRCDIR/..
+PACKAGES=$ROOTDIR/packages
+
+PROJFS=$SRCDIR/ProjFS.Linux
+
+# TODO(Linux): use pkg-config to look for installed libprojfs;
+#              if not found, retrieve latest stable from repo, build
+#              into $ROOTDIR/BuildOutput/ProjFS.Linux/Native with
+#              given $CONFIGURATION
+
+# If we're building the Profiling(Release) configuration, remove Profiling() for building .NET code
+if [ "$CONFIGURATION" == "Profiling(Release)" ]; then
```

Comment:
I think this is pretty Xcode-specific, you can probably drop it

Reply:
Sure, will do.